### PR TITLE
Admin merchant dashboard refactor

### PR DESCRIPTION
**Pull request by jdcarey128**
This PR: 
- Changes admin/merchant and merchant dashboard to display correct value and item count for pending orders 
- Changes order names in admin dashboard to match user name (not order name)
- Adds a format_price helper 
- Changes 'unfulfilled' to 'fulfilled' in fulfilled? item method 
- Refactors things here and there
- ALL tests passing at 99.9% coverage

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ module ApplicationHelper
   def format_date(input)
     input.strftime("%m/%d/%Y")
   end
+
+  def format_price(input)
```
Comment (smcunning) [inline app/helpers/application_helper.rb:7]: Nice! Gonna use this one in a refactor.

**Review by smcunning — APPROVED** [review]
LGTM! Thanks for squashing these.

**Review by katemorris — APPROVED** [review]
Looks great! Love the refactors!

**Review by taphill — APPROVED** [review]
Nice job!

**Pull request merged**